backend/emailer/views.py

```diff
@@ -0,0 +1,60 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.http import HttpResponse
+from django.shortcuts import redirect
+from django.utils.encoding import force_text
+from django.views import View
+
+from api.models import User
+from .models import EmailLog
+from .tokens import account_activation_token
+
+
+def send_activation_email(user):
+    recipient = user.email
+    token = account_activation_token.make_token(user)
+    activate_url = urljoin(settings.HOST_ROOT_URL, 
+                            'e/activate/{uid}/{token}/'.format(uid=user.pk, token=token))
+    
+    message = 'Welcome to Cultidate!\n\nTo activate your account please click here: {url}\n\nBest regards,\nCultidate Team'.format(url=activate_url)
+    
+    try:
+        send_mail(
+            '[Cultidate] Activate your account',
+            message,
+            settings.EMAIL_HOST_USER,
+            [recipient],
+            fail_silently=False
+        )
+        sent = True
+    except:
+        sent = False
+    
+    EmailLog.objects.create(user=user,
+                            sender=settings.EMAIL_HOST_USER,
+                            recipient=recipient,
+                            success=sent)
+
+    return sent
+
+
+class ActivateAccountView(View):
```

Comment:
There is a class-based view called [RedirectView](https://docs.djangoproject.com/en/2.1/ref/class-based-views/base/#django.views.generic.base.RedirectView). Your implementation is totally fine, though.